packages/google_sign_in/google_sign_in_web/lib/src/gis_client.dart

```diff
@@ -12,7 +12,6 @@ import 'package:web/web.dart' as web;
 
 import 'button_configuration.dart'
     show GSIButtonConfiguration, convertButtonConfiguration;
-import 'people.dart' as people;
```

Comment:
I think this `people.dart` file is now dead code, you can maybe delete it and its tests?